src/tcti-tabrmd.c

```diff
@@ -433,7 +433,7 @@ typedef struct {
     GBusType type;
 } bus_name_type_entry_t;
 
-const static bus_name_type_entry_t bus_name_type_map[] = {
+static const bus_name_type_entry_t bus_name_type_map[] = {
```

Comment:
This is a good fix. I remember the order of these two qualifiers being unimportant but it looks like C is now obsoleting storage class specifiers anywhere in declarations other than at their beginning.

Reply:
The compiler complained and warned about "const static". Changing to "static const" stopped the warning.